modules/API/API.js

```diff
@@ -310,6 +314,29 @@ function initCommands() {
 
             APP.store.dispatch(sendTones(tones, duration, pause));
         },
+        'set-assumed-bandwidth-bps': value => {
+            logger.debug('Set assumed bandwidth bps command received', value);
+
+            if (isNaN(value)) {
+                logger.error('Assumed bandwidth bps must be a number.');
+
+                return;
+            }
+
+            if (value > ASSUMED_BANDWIDTH_BPS.MAX_VALUE) {
```

Comment:
Same as in the other PR, do we want to have  the limit in (now) 3 places?

Reply:
👍

Reply:
Let's reopen the can o works :-) LJM no longer checks for a max value, and the JVB doesn't seem to have one either, so I'd say we drop this  check.